lldb/source/Target/Process.cpp

```diff
@@ -4248,7 +4248,21 @@ bool Process::ProcessEventData::ShouldStop(Event *event_ptr,
   return still_should_stop;
 }
 
+bool Process::ProcessEventData::ForwardEventToPendingListeners(Event *event_ptr) {
+  // STDIO and the other async event notifications should always be forwarded.
+  if (event_ptr->GetType() != Process::eBroadcastBitStateChanged)
+    return true;
+
+  // For state changed events, if the update state is one, we are handling 
+  // this on the private state thread.  We should wait for the public event.
+  return m_update_state == 1;
```

Comment:
You say:
> // if the update state is one, we are handling this on the private state thread. We should wait for the public event.

Yet:
> return m_update_state == 1;

I'm confused because this would mean we should forward events to pending listeners if the update state is one. 

I believe you meant:
```suggestion
  // For state changed events, if the update state is zero, we are handling 
  // this on the private state thread.  We should wait for the public event.
  return m_update_state == 1;
```

Reply:
Yup, just wrote the wrong thing in the comment.